forge-core/src/main/java/forge/card/CardFace.java

```diff
@@ -51,7 +47,7 @@ public enum FaceSelectionMethod { //
     private List<String> abilities = null;
     private List<String> staticAbilities = null;
     private List<String> triggers = null;
-    private List<String> draftActions = null;
+    private List<String> draftActions = Lists.newArrayList();
```

Comment:
why would all instances need empty list for this 0.1% as default now?
not sure what the JVM will make of that memory wise, but feels like bad design anyway